### PR TITLE
Remove no needed properties odata.type and subscriptionName in create Cloudpc onpremisesconenctions

### PR DESCRIPTION
**Pull request by Emma0727**
Fixed the example for create Cloudpc onpremisesconenctions. The example is wrong and An issue is raised for this.
[create Cloudpc onpremisesconenctions'](https://github.com/microsoftgraph/microsoft-graph-docs/issues/19561)
Remove no needed properties odata.type and subscriptionName in create Cloudpc onpremisesconenctions's body, they are not needed.
No need to change the CSDL file, as no properties changes.

Road map ADO: https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/15967

### CODE REVIEW

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md [hunk 1]
```diff
@@ -85,11 +85,9 @@ POST https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/onPremise
 Content-Type: application/json
 
 {
-  "@odata.type": "#microsoft.graph.cloudPcOnPremisesConnection",
   "displayName": "test-canary-02",
   "type": "hybridAzureADJoin",
   "subscriptionId": "0ac520ee-14c0-480f-b6c9-0a90c585ffff",
-  "subscriptionName": "CPC customer 001 test subscription",
```
Comment (RuiHou105) [inline api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:92]: Why remove it

**Comment by RuiHou105** [comment]
Please create a roadmap workitem and attach the workitem number here.

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md [hunk 2]
```diff
@@ -85,11 +85,9 @@ POST https://graph.microsoft.com/beta/deviceManagement/virtualEndpoint/onPremise
 Content-Type: application/json
 
 {
-  "@odata.type": "#microsoft.graph.cloudPcOnPremisesConnection",
```
Comment (RuiHou105) [inline api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:88]: Why remove it?

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:92]: Okay got it. The request body shouldn't have this line.

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md (hunk 2 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:88]: Okay got it. The request body shouldn't have this line.

File: api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md (hunk 1 above)
Comment (RuiHou105) [reply api-reference/beta/api/virtualendpoint-post-onpremisesconnections.md:92]: Read only.

**Comment by Emma0727** [comment]
https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/15967
this is the road map ADO

**Comment by Emma0727** [comment]
> Please create a roadmap workitem and attach the workitem number here.

attached

**Review by RuiHou105 — APPROVED** [review]

**Review by Danielabom — APPROVED** [review]
@Emma0727 , thank you for working on this PR. 
Based on the roadmap, there won't be changes to the schema. Changes look good.

**Comment by Danielabom** [comment]
@Emma0727, friendly reminder to please add the "ready to merge" label as soon as you are ready to publish.

**Pull request merged**